utils/get_shop_products/index.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+import { SHOP_PRODUCTS_ENDPOINT } from '../../constants';
+
+export const get_shop_products = async () => {
+  try {
+    const {
+      data: { Products },
+    } = await axios.get(SHOP_PRODUCTS_ENDPOINT);
+    // console.log(Products);
+    return Products;
+  } catch (error) {
+    console.log(error.message);
```

Comment:
Can we have a way where we can let the user's on the screen know that there has been some error, rather than just console logging it. If I click on something which is performing an `http` request, If there is an error, how will I know about it. What say @Kratika0907

Reply:
Exactly what i was thinking , but for that we should better restructure the redux for handling and keeping track of network errors